ts/src/base/Exchange.ts

```diff
@@ -2339,6 +2339,51 @@ export default class Exchange {
         return result;
     }
 
+    async fetchWebEndpoint (method, endpointMethod, returnAsJson, startRegex = undefined, endRegex = undefined) {
```

Comment:
@ttodua This method has no other usage except inside gemini, so I would probably move it there for now.

Reply:
as I've commented about the thought to include that in base, do you still want to move it into gemini specifically?

Reply:
@carlosmiei if you confirm or not the above comment

Reply:
build passes, irrelevant failures